src/raw/client.rs

```diff
@@ -491,6 +492,28 @@ impl Client {
         plan.execute().await
     }
 
+    pub async fn coprocessor(
+        &self,
+        copr_name: String,
+        copr_version_req: String,
```

Comment:
Should we maybe use `semver::VersionReq` here so that we can be sure that the user passes a valid string?

Reply:
Good idea!